public/assets/questions.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
```

Comment:
I really think we should be using the handlebars from now on, that way you don't need to put the nav bar every time and it will be simpler.

use partials/world1/world1-block

and add it to index.handlebars like how the other things are used.